trajectory_msgs/JointTrajectory trajectories
float32 speed_factor
---
std_msgs/Bool complete_execution